packages/flutter_tools/lib/src/test/flutter_platform.dart

```diff
@@ -380,8 +382,9 @@ class _Compiler {
   }
 }
 
-class _FlutterPlatform extends PlatformPlugin {
-  _FlutterPlatform({
+/// The flutter test platform used to integrate with package:test.
+class FlutterPlatform extends PlatformPlugin {
```

Comment:
Consider adding docs for the public fields/methods.

Reply:
I'd like to significantly refactor this configuration so these will likely move around a bit over the next few weeks. I will ensure each field is documented once moved